Makefile

```diff
@@ -0,0 +1,13 @@
+# Makefile for Truss.
+#
+# Build native Truss by default.
+default:
+	$(MAKE) -C truss
+
+# Run the go tests and the truss integration tests
+test:
+	go test -v ./...
```

Comment:
This runs all the tess, including vendored tests, and that's pretty annoying. Can this be changed to maybe:

```
go test -v ./deftree/... ./gengokit/... ./gendoc/... ./truss/...
```

So that it only runs the tests for modules of truss?